frontend/__mocks__/components/Localized.tsx

```diff
@@ -1,15 +1,6 @@
-import { ReactNode, cloneElement, isValidElement } from "react";
+import { cloneElement, isValidElement, ReactNode } from "react";
```

Comment:
Heh, good question :sweat_smile: I think it might just be me first removing the import and then later needing it again, but it might indeed also be Prettier sorting it at the end?